src/Microsoft.Azure.SignalR/ServerConnections/ServiceConnectionContext.cs

```diff
@@ -35,6 +35,10 @@ internal class ServiceConnectionContext : ConnectionContext,
             readerScheduler: PipeScheduler.ThreadPool,
             useSynchronizationContext: false);
 
+        private readonly TaskCompletionSource<bool> _connectionEndTcs = new TaskCompletionSource<bool>(TaskContinuationOptions.RunContinuationsAsynchronously);
```

Comment:
>TaskContinuationOptions [](start = 103, length = 23)

`TaskCreationOptions`, as pointed out by Brennan, they are different although the value is the same